CREATE TABLE produkt

```diff
@@ -0,0 +1,70 @@
+CREATE TABLE produkt 
+(id_produktu bigserial PRIMARY KEY,
+ nazwa VARCHAR(50) UNIQUE NOT NULL,
+kategoria VARCHAR(50) NOT NULL,
+ilosc SMALLINT NOT NULL,
+cena NUMERIC(7,2) NOT NULL,
+rozmiar VARCHAR(20) NOT NULL,
+polozenie VARCHAR(20) NOT NULL,
+data_modyfikacji timestamp NOT NULL)
+
+
+CREATE TABLE zespol 
+(id_zespolu serial PRIMARY KEY,
+ kierownik VARCHAR(50) NOT NULL,
+zmiana VARCHAR(50) NOT NULL)
+
+
+CREATE TABLE klient 
+(id_klienta bigserial PRIMARY KEY,
+ imie VARCHAR(50) NOT NULL,
+nazwisko VARCHAR(50) NOT NULL,
+email VARCHAR(50) UNIQUE NOT NULL CHECK (email ~* '^[A-Za-z0-9._%-]+@[A-Za-z0-9.-]+[.][A-Za-z]+$'),
+telefon VARCHAR(20) ,
+adres VARCHAR(50) NOT NULL,
+met_platnosci VARCHAR(50) NOT NULL,
+data_utworzenia timestamp NOT NULL)
+
+
+CREATE TABLE pracownik 
+(id_pracownika bigserial PRIMARY KEY,
+ id_zespolu INTEGER REFERENCES zespol (id_zespolu) NOT NULL,
+pozycja VARCHAR(50) NOT NULL,
+imie VARCHAR(50) NOT NULL,
+nazwisko VARCHAR(50) NOT NULL,
+data_utworzenia timestamp NOT NULL)
+

```

Comment:
I'd add 'email' column to Pracownik, because Pracownik will probably use email for signing in to the system